main.js

```diff
@@ -162,6 +154,44 @@ function renderDetails(item) {
   ])
 }
 
+function renderCartItem(item) {
+  return createElement('div', { class: 'col-6 offset-3 cart-col' }, [
+    createElement('img', { class: 'cart-img', src: item.imageUrl, alt: 'Card image cap' }, []),
+    createElement('h5', { class: 'card-title cart' }, [item.brand]),
+    createElement('p', { class: 'card-text item-name cart' }, [item.name]),
+    createElement('p', { class: 'card-text cart' }, [currencyFormat(item.price)])
+  ])
+}
+
+function renderCartTotal(cart) {
+  var total = cart.items.reduce((acc, item) => {
+    return acc + item.price
+  }, 0)
+  return createElement('div', { class: 'col-6 offset-3' }, [
+    createElement('div', { class: 'text-right cart-total' }, ['Total: ' + currencyFormat(total)])
+  ])
+}
+
+function renderCartItemCount(cart) {
+  return createElement('div', { class: 'col-6 offset-3' }, [
+    createElement('div', { class: 'cart-item-count text-right cart-total' }, ['Items: ' + cart.items.length])
+  ])
+}
+
+function renderCartSummary(cart) {
+  var $summary = createElement('div', { class: 'container' }, [
+    createElement('h2', { class: 'cart-header text-center' }, ['Cart Summary'])
+  ])
+  app.cart.items.forEach((item) => {
```

Comment:
This might require some changes to your CSS (maybe not), but consider using `Array.prototype.map` here instead. When we move on to React, `forEach` won't be a realistic option.
```js
var $summary = createElement('div', { class: 'container' }, [
  createElement('h2', { class: 'cart-header text-center' }, ['Cart Summary']),
  createElement('div', {}, app.cart.items.map(item => renderCartItem(item))),
  renderCartItemCount(cart),
  renderCartTotal(cart),
  createElement('button', { class: 'btn btn-primary', id: 'cart-continue-shopping' }, ['Continue Shopping'])
])
```